src/database.py

```
@@ -0,0 +1,130 @@
+import sqlite3
+from pathlib import Path
+from typing import Any, List, Dict, Optional
+from config import DB_PATH
+
+def _init_db_pragmas(conn):
+    conn.execute("PRAGMA journal_mode=WAL;")
+    conn.execute("PRAGMA wal_checkpoint(FULL);")
+    conn.execute("PRAGMA read_uncommitted = TRUE;")
+
+class ChatDBClient:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)